samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -7,9 +7,11 @@
 using Microsoft.SemanticKernel;
 using Microsoft.SemanticKernel.AI;
 using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
 using SemanticKernel.Service.Config;
 using SemanticKernel.Service.Model;
 using SemanticKernel.Service.Skills;
+using SemanticKernel.Service.Skills.OpenAPI.Authentication;
```

Comment:
Do we need both SemanticKernel.Service.Skills.OpenAPI.Authentication and Microsoft.SemanticKernel.Skills.OpenAPI.Authentication ?

Reply:
The former is the local copilot chat namespace -- decided to keep the GitHub skill confined to copilot chat